selfdrive/car/interfaces.py

```diff
@@ -4,14 +4,30 @@
 from common.kalman.simple_kalman import KF1D
 from common.realtime import DT_CTRL
 from selfdrive.car import gen_empty_fingerprint
+from selfdrive.controls.lib.vehicle_model import VehicleModel
 
 GearShifter = car.CarState.GearShifter
 
 # generic car and radar interfaces
 
 class CarInterfaceBase():
-  def __init__(self, CP, CarController):
-    pass
+  def __init__(self, CP, CarController, CarState, get_can_parser, get_cam_can_parser=None):
```

Comment:
get_can_parser and get_cam_can_parser can just be static methods of the CarState class.
No need to do it now, but a good clean up for later.

Reply:
Indeed, it would be better to have the parent class having the same init API as the subclasses, so we can avoid
https://github.com/commaai/openpilot/pull/1131/files#diff-93874011152ca529debcfce500178461R16